docs/kyma/docs/032-gs-cluster-installation.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Cluster Kyma installation
+type: Getting Started
+---
+
+## Overview
+
+This Getting Started guide instructs developers to quickly deploy Kyma on a cluster. Kyma installs on a cluster using a proprietary installer based on a Kubernetes operator.
+
+The document provides the prerequisites, the instructions on how to install Kyma and verify the deployment, and the troubleshooting tips.
```

Comment:
I would say it like this:

"The document provides prerequisites, instructions on how to install Kyma and verify the deployment, as well as troubleshooting tips."

Reply:
Same here